src/Google.Api.Gax.Grpc/MonitoredResourceBuilder.cs

```diff
@@ -0,0 +1,88 @@
+﻿/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax.Grpc
+{
+    /// <summary>
+    /// Helper methods to build a <see cref="MonitoredResource"/> instance.
+    /// See the <a href="https://cloud.google.com/logging/docs/api/v2/resource-list">
+    /// Monitored Resource List</a> for details.
+    /// </summary>
+    public static class MonitoredResourceBuilder
+    {
+        /// <summary>
+        /// An instance of a "global" resource, with <see cref="MonitoredResource.Type"/>
+        /// set to "global", and an empty set of <see cref="MonitoredResource.Labels"/>.
+        /// </summary>
+        public static MonitoredResource GlobalResource =>
```

Comment:
Possibly add a comment to show why it's not just a `{ get; } = ...`... because this is mutable.